include/rosdiscover-clang/Value/String.h

```diff
@@ -16,13 +16,17 @@ class StringLiteral : public virtual SymbolicString {
   }
 
   void print(llvm::raw_ostream &os) const override {
-    os << "\"" << literal << "\"";
+    os << toString();
+  }
+  std::string toString() const override {
+    return "'\"" + literal + "\"'";
```

Comment:
this is fine, but for the future, look into using `fmt::format` (https://github.com/fmtlib/fmt)

Reply:
fixed